components/openapi/src/main/scala/pl/touk/nussknacker/openapi/discovery/SwaggerOpenApiDefinitionDiscovery.scala

```diff
@@ -44,7 +44,7 @@ class SwaggerOpenApiDefinitionDiscovery(implicit val httpBackend: SttpBackend[Fu
       FileUtils.readFileToString(new File(discoveryUrl.getPath), StandardCharsets.UTF_8)
     } else {
       Await
-        .result(basicRequest.get(Uri(discoveryUrl.toURI)).send(), 20 seconds)
+        .result(basicRequest.get(Uri(discoveryUrl.toURI)).send(httpBackend), 20 seconds)
```

Comment:
I see that now, backend must be passed explicitly. If their pick this approach, IMO we should remove implicit variables modifier. Otherwise it can cause confusions how this backend is used - why it is used implicitly and ath the same time, passed explicitly.

Reply:
Yes, implicit SttpBackend passing is deprecated, but I'm not sure about passing it explicitly in our code - from what I see we do pass various contexts as implicits, so that would make Sttp special for us.

Also, I wanted this change to be as small as possible, it already touches enough files.

Reply:
Ok, let it be that way